src/es/components/organisms/filterList/FilterList.js

```diff
@@ -54,17 +54,23 @@ export default class FilterList extends Shadow() {
    * @param {import('../../controllers/events/Events.js').ListFilterItems} data
    */
   renderHTML (data) {
-    this.html = ''
-    if (!data || !Array.isArray(data.items)) return
-    this.html = /* html */`
-      <ul class="list-items">
-        ${Array.from(new Set(
-          // @ts-ignore
-          data.items
-        )).map(name => /* html */`
```

Comment:
@MrBuggy , the problem with the "," was that we fed this.html an array with .map, array.reduce solves the problem.